.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
- [ ] It would be great if you add the following commands in the Getting Started section to make your readme descriptive and professional 👍

```
git clone your-repo
cd your-repo
npm install
npm run build
npm start
```

Reply:
thank you @DammyShittu . Changes done

Reply:
I appreciate your contribution, although the requirements are not so specific with respect to the readme file